tests/scripts/run-notebook-test.sh

```diff
@@ -75,7 +75,7 @@ for env in base /home/ec2-user/anaconda3/envs/*; do
     $ENV_PYTHON -m pip install --upgrade pip
     $ENV_PYTHON -m pip install "$TARBALL_DIRECTORY/sagemaker.tar.gz"
 
-    sudo -u ec2-user -E sh -c 'source /home/ec2-user/anaconda3/bin/deactivate'
+    sudo -u ec2-user -E sh -c 'conda deactivate'
```

Comment:
I believe notebbok-tests are failing due to this? Can you revert this change?

Reply:
Done. Could you execute tests again?